### DISCUSSION TITLE
Why does adding word to dictionary not work?

### DISCUSSION THREAD

**Discussion by lwrage** [valentjn/vscode-ltex#281] [Q&A] [answered]
I have added word to the dictionary, but it is still marked as a spelling error. The word is added to the dictionary in .vscode/ltex.dictionary.en-US.txt. Is there some other configuration necessary to actually use the external dictionary file? This is with LTeX 9.0.0, vscode 1.54.3)

**Comment by valentjn** [comment, marked as answer]
No, it should work out-of-the-box (see [the docs](https://valentjn.github.io/vscode-ltex/docs/advanced-usage.html#external-setting-files)). Please [file a bug report](https://valentjn.github.io/vscode-ltex/docs/contributing-code-issues.html#how-to-report-bugs) with all the necessary info. Otherwise, it's hard to see what went wrong.

**Reply by lwrage** [reply]
It started working after a restart. It worked immediately on another computer.

**Reply by valentjn** [reply]
Weird. If it happens again and you can reproduce it, open an issue.
